javascript/builtins/RegExp.json

```
@@ -426,6 +426,57 @@
             }
           }
         },
+        "hasIndices": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp/hasIndices",
+            "support": {
+              "chrome": {
+                "version_added": "91"
```

I was wondering how this was coming along and I couldn't figure it out from the bug links. I tested it though and found that it's available in 90 (beta now, but stable in a few days), not 91.

```suggestion
                "version_added": "90"
```

---

Done.